src/main/kotlin/com/github/devcordde/devcordbot/listeners/DevmarktRequestUpdater.kt

```diff
@@ -91,4 +104,100 @@ class DevmarktRequestUpdater(
 
         event.jda.httpClient.newCall(request).execute()
     }
+
+    /**
+     * Reacts if a message is received
+     */
+    @SubscribeEvent
+    fun onMessageReceived2(event: GuildMessageReceivedEvent) {
```

Comment:
onReceiveDenyMessage

Reply:
To my defence, I thought, I'd changed that already.